src/scripts/main.js

```diff
@@ -21,7 +21,39 @@
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  document.body.appendChild(ul);
+
+  for (const key1 in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key1;
+
+    ul.appendChild(li);
+
+    if (typeof data[key1] === 'object') {
+      for (const key2 in data[key1]) {
+        if (key2 !== undefined) {
+          const li1 = document.createElement('li');
+
+          li1.textContent = key2;
+
+          li.appendChild(li1);
+
+          for (const key3 in data[key1][key2]) {
+            if (typeof key3 === 'string') {
+              const li2 = document.createElement('li');
+
+              li2.textContent = key3;
+
+              li1.appendChild(li2);
+            }
+          }
+        }
+      }
+    }
+  }
```

Comment:
Consider using recursion instead of nested loops to handle nested objects. This will make the code cleaner and more efficient for nested structures.